src/tx-format/transaction.md

```diff
@@ -131,15 +131,25 @@ Creates a contract with contract ID as computed [here](../identifiers/contract-i
 ## TransactionMint
 
 The transaction is created by the block producer and is not signed. Since it is not usable outside of block creation or execution, all fields must be fully set upon creation without any zeroing.
-
-| name           | type                         | description                                          |
-|----------------|------------------------------|------------------------------------------------------|
-| `txPointer`    | [TXPointer](./tx-pointer.md) | The location of the `Mint` transaction in the block. |
-| `outputsCount` | `uint8`                      | Number of outputs.                                   |
-| `outputs`      | [Output](./output.md)`[]`    | List of outputs.                                     |
+This means that the transaction ID must also include the correct `txPointer` value, not zeroed out.
+
+| name           | type                         | description                                               |
+|----------------|------------------------------|-----------------------------------------------------------|
+| `mintAmount`   | `uint64`                     | The amount of funds being minted within this transaction. |
+| `mintAssetId`  | `byte[32]`                   | The asset ID of funds minted within this transaction.     |
+| `gasLimit`     | `uint64`                     | Gas limit for transaction                                 |
```

Comment:
If we don't have gas price, it means mint transaction is free. So maybe it is better if the gas limit is part of the `ConsensusParams` instead of a customizable value?

Reply:
Yeah that makes sense, we'd essentially be saying there's a max limit of execution time a producer is allowed to spend on their coinbase processing agreed upon by all chain participants.

Reply:
removed all scripting from mint tx